### PR TITLE
Standardized indentation

### PR DESCRIPTION
**Pull request by tiagodenoronha**

### CODE REVIEW

**Comment by ajcvickers** [comment]
@tiagodenoronha We typically indent to 'tab' stops (i.e. every four spaces) rather than aligning dots. That being said, some of the indentation here does seem dubious--I'll let @divega decide if it should be changed.

File: entity-framework/ef6/querying/related-data.md [hunk 1]
```diff
@@ -54,18 +54,18 @@ using (var context = new BloggingContext())
                        .Include(b => b.Posts.Select(p => p.Comments))
```
Comment (divega) [inline entity-framework/ef6/querying/related-data.md]: This one is still not aligned.

Comment (tiagodenoronha) [reply entity-framework/ef6/querying/related-data.md]: Wait, it isn't? Is it now?
Or do you want the "Select" below the "Posts"?

**Comment by divega** [comment]
Thank you @tiagodenoronha for your contribution.

**Comment by tiagodenoronha** [comment]
Thank you for accepting my first pull request on github!

**Pull request merged**